src/main/java/com/minecolonies/items/ItemAchievementProxy.java

```diff
@@ -0,0 +1,15 @@
+package com.minecolonies.items;
+
+/**
+ */
+public class ItemAchievementProxy extends AbstractItemMinecolonies
+{
+
+    public ItemAchievementProxy(String name)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)